radixdlt-java/src/main/java/com/radixdlt/fees/FeeTable.java

```diff
@@ -50,6 +55,15 @@ public UInt256 minimumFee() {
 		return this.feeEntries;
 	}
 
+	public UInt256 feeFor(Atom atom) {
```

Comment:
Might be worth considering changing the name of the argument to `atomWithoutFees` to make it clear that this will not correctly calculate the fee for an atom that already has fees included.